### DISCUSSION TITLE
github student developer pack

### DISCUSSION THREAD

**Discussion by gandham-sai-gopi** [github/india#360] [GitHub Platform]
hello all ,shall anyone please guide me how to get github free developer pack 
please tell us the detailed steps to be followed 
and how to use that pack

**Comment by TheSuranaverse** [comment]
@gandham-sai-gopi 
To get the **STUDENT DEVELOPER PACK** you need to first add "Microsoft Azure" as a skill on LinkedIn & then apply for it.
A detailed process along with the images is provided in the link attached below.
Link: https://github.com/Neklaustares-tPtwP/Microsoft-Future_Ready_Talent-Internship/tree/master/Hot%20Queries%20%26%20Solution

Also, it will take at least 3 days to process your request.
To get personal assistance on how to receive it you can connect with our community here: https://discord.gg/drQ7ySCdX2
We all help each other & clear everyone's doubt on a personal basis there. 
If this answers your question, please mark it as answered.

**THANK YOU!**

**Comment by FRTMentor** [comment]
Please reach out the GitHub support team.
